src/dbnode/storage/bootstrap.go

```diff
@@ -329,26 +305,17 @@ func (m *bootstrapManager) bootstrap() error {
 		})
 	}
 
-	logFields := []zapcore.Field{
-		zap.Int("numShards", len(uniqueShards)),
-	}
-	m.log.Info("bootstrap started", logFields...)
-
+	m.instrumentation.bootstrapStarted(instrCtx, len(uniqueShards))
```

Comment:
Consider moving those methods to `instrCtx`, so that you can just call `instrCtx.bootstrapStarted(len(uniqueShards))`.